swagger_to/go_server.py

```diff
@@ -561,15 +573,16 @@ def _escaped_str(text: str) -> str:
 {% if fielddef.name not in typedef.required %}
 {% set json_tags = json_tags + ',omitempty' %}
 {% endif %}
-    {{ fielddef.name }} {{ field_type[fielddef] }} `json:{{ json_tags|escaped_str }}`{% if loop.last %}}{% endif %}
```

Comment:
If I remember correctly, this was necessary if we want to handle nested objects in the future.

Was it only a cosmetics change or there was a deeper reason why you moved the curly brackets on the new line?

If it's only cosmetics: could you undo it in the pull request to keep it a bit easier to trace. After merging this pull request, I'll create an additional one to fix this issue. What do you think?